texttospeech/cloud-client/src/main/java/com/example/texttospeech/SynthesizeText.java

```diff
@@ -35,41 +35,42 @@
 import net.sourceforge.argparse4j.inf.Namespace;
 
 /**
- * Google Cloud TextToSpeech API sample application.
- * Example usage: mvn package exec:java -Dexec.mainClass='com.example.texttospeech.SynthesizeText'
- *                                      -Dexec.args='text "hello"'
+ * Google Cloud TextToSpeech API sample application. Example usage: mvn package exec:java
+ * -Dexec.mainClass='com.example.texttospeech.SynthesizeText' -Dexec.args='--text "hello"
+ * "telephony-class-application"'
  */
 public class SynthesizeText {
 
   // [START tts_synthesize_text]
   /**
    * Demonstrates using the Text to Speech client to synthesize text or ssml.
+   *
    * @param text the raw text to be synthesized. (e.g., "Hello there!")
    * @throws Exception on TextToSpeechClient Errors.
    */
-  public static void synthesizeText(String text)
-      throws Exception {
+  public static void synthesizeText(String text, String effectsprofile) throws Exception {
```

Comment:
You'll need to create a 2nd method called `synthesizeTextWithAudioProfile`

See: https://github.com/GoogleCloudPlatform/python-docs-samples/pull/1538/files